xapian-letor/include/xapian-letor/featuremanager.h

```diff
@@ -67,34 +66,18 @@ class XAPIAN_VISIBILITY_DEFAULT FeatureManager {
      */
     std::map<int,double> transform(const Document &doc, double &weight_);
 
-    /** Calculates ranklist (0 qid:10032 1:0.130742 2:0.000000 ... 18:0.750000 19:1.000000 #docid = 1123323)
-     *  for an mset retrieved from a query. Feature 19 is the BM25 weight.
+    /** Returns a vector of FeatureVectors for each document in the MSet for a given query.
      *
-     *  @ param  train      Set to 1 while training.
+     *  @ param  mset      MSet for which the vector<FeatureVector> is to be returned
      */
-    Xapian::RankList create_rank_list(const Xapian::MSet & mset, std::string & qid, bool train);
-
-    /** Loads up a qrel file.
-     *  @return a map: < qid <docid, relevance_judgement> >
-     */
-    map<string, map<string,int> > load_relevance(const std::string & qrel_file);
-
-    /** Creates instance of FeatureVector class that stores features_array, relevance label and corresponding document id
-     *  @return a FeatureVector instance containing features_array, relevance label and corresponding document id
-     */
-    Xapian::FeatureVector create_feature_vector(map<int,double> fvals, int &label, Xapian::docid & did);
+    std::vector<Xapian::FeatureVector> create_vec_fvec(const Xapian::MSet & mset);
```

Comment:
I put the name as `create_vec_fvec` instead of `create_feature_vectors` since FeatureManager has a closely named method called `create_feature_vector` (which will actually be removed in the Features PR). But still, `create_vec_fvec` isn't very intuitive. Will make the change.